frontend/src/components/Utils/time.js

```diff
@@ -58,3 +60,22 @@ export function timestampToFormatted(msTimestamp, timezone = 'America/New_York')
   let formatted = date.toLocaleString('en-US', formatOptions);
   return formatted;
 }
+
+/**
+ * Returns all the time zones in a country (in pretty format).
+ * 
+ * @param {string} countryName The name of the country for which to get the time zones.
+ * @returns The list of time zones in the provided country.
+ */
+export function timezonesForCountry(countryName) {
+  let zones;
+  try {
+    const countryCode = countryCodes[countryName];
+    zones = moment.tz.zonesForCountry(countryCode);
+  } catch (e) {
+    zones = moment.tz.names(); // List of all timezones.
+  }
+  return zones.map(e => {
+    return e.replace(/[_]/g, ' ');
+  });
```

Comment:
Why would a timezone like "America/New_York" need to be replaced with "America/New York"? Is it just for styling or is this for something else?

Reply:
For styling, since this goes inside the dropdown.